tests/test_integration/test_eval_loop_integration.py

```
@@ -439,3 +439,88 @@ class _FakeJudgeClient:
         judge = result["scoring_details"]["judge"]
         assert judge["total"] is None
         assert "judge exploded" in judge["error"]
+
+
+class _JudgeScoreEnv(EvalEnvironment):
+    """Env whose verify() requests the built-in judge_score path (no _judge_fn)."""
+
+    name = "mock_judge_score"
+
+    def __init__(self, n):
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)